adam/adam_6224.py

```diff
@@ -0,0 +1,628 @@
+
+# -*- coding: utf-8 -*-
+#
+# This file is part of the dev-solaris-adam project
+#
+#
+#
+# Distributed under the terms of the LGPL license.
+# See LICENSE.txt for more info.
+
+
+__all__ = ["ADAM6224", "main"]
+
+# PyTango imports
+import tango
+from tango import DebugIt
+from tango.server import run
+from tango.server import Device, DeviceMeta
+from tango.server import attribute, command
+from tango.server import class_property, device_property
+from tango import AttrQuality, AttrWriteType, DispLevel, DevState
+# Additional import
+# PROTECTED REGION ID(ADAM6224.additional_import) ENABLED START #
```

Comment:
Please remove all comments for `PROTECTED REGIONS` as they won't be needed anymore.